content/collections/data/en/automated-tasks-in-data-assistant.md

```diff
@@ -0,0 +1,100 @@
+Automated Tasks are part of Amplitude’s [Data Assistant](/docs/data/use-ai-data-assistant) feature. Automated tasks let data governors define conditions and actions that the system then performs automatically. It automates repetitive clean-up workflows by taking recommendations provided by the Data Assistant and transforming those recommendations into automatic actions. For example, an automated task can automatically delete events if they have been unused for 90 days. 
+
+After they have been created, automated tasks run daily. 
+
+Automated tasks is only available on Enterprise plans. Go to the [pricing](https://amplitude.com/pricing) page for more details.
+
+{{partial:admonition type="note" heading=""}}
+Automated tasks are only available for users with manager or admin permissions. Contact your administrator if you can't access Automated tasks as an option for Data Assistant.
+{{/partial:admonition}}
+
+You can access your automated tasks from the Data Assistant by going to *Data > Assistant > Automated Tasks*. 
+If you have active automations, those are displays along with a current count of the events affected by that specific automation. If you don't have any active automations, you'll have the opportunity to set them up. 
+
+By default, automated tasks operate across all projects within a workspace. You can, however, specify for the automation to only search through a single project.
+
+## Types of automated tasks
+
+You can select one, or any, of the following types of tasks to automate: 
+
+* Clean up stale events
+* Clean up single-day events
+* Clean up unused events
+
+### Clean up stale events
+This tasks removes events that haven't experiences any recent volume. This indicates that the events aren't being ingested and are no longer of value. The task inspects your organization for events that have a `last seen` date in a configurable number of days. For example, events that haven't been ingested in 90 days.
+
+Historical charts or data aren't affected. 
+
+### Clean up single-day events
+This task removes accidental or one-time test events that can negatively impact or clutter up your taxonomy. The task inspects your organization for events that:
+
+* Have the same first seen and last seen dates and;
+* That date is more than a configuraable number of days before the inspection date. (For example, 90 days before the inspection date.)
+
+When the automated task finds events that match those criteria, it automatically schedules those events for deletion. All people using those events are notified through email or Slack about the impending deletion. Anywhere the events are being used (for example, in a chart), a banner appears notifying users about the upcoming deletion. If anyone wants to keep the event, they can do so through the email or banner notification. If no one elects to keep the event within 30 days, the event is deleted. Deleted events no longer appear in the event drop-down menu and are blocked from future ingestion.
```

Comment:
```suggestion
When the automated task finds events that match those criteria, it schedules those events for deletion. All people using those events are notified through email or Slack about the impending deletion. Anywhere the events are being used (for example, in a chart), a banner appears notifying users about the upcoming deletion. If anyone wants to keep the event, they can do so through the email or banner notification. If no one elects to keep the event within 30 days, the event is deleted. Deleted events no longer appear in the event drop-down menu and are blocked from future ingestion.
```
- Removed "automatically", feels unnecessary.
- Can we be more specific about which "people" using those events? Amplitude users in the org?